main/control/user.py

```diff
@@ -24,7 +24,8 @@
 @app.route('/admin/user/')
 @auth.admin_required
 def user_list():
-  user_dbs, user_cursor = model.User.get_dbs(email=util.param('email'))
+  user_dbs, user_cursor, prev_cursor =\
+      model.User.get_dbs(email=util.param('email'), prev_cursor=True)
```

Comment:
Well the normal would be the `next_cursor`, `prev_cursor`.. but what about that case:

```
user_dbs, next_cursor, prev_cursor = model.User.get_dbs(prev_cursor=True)
...
company_dbs, next_cursor, prev_cursor = model.Company.get_dbs(prev_cursor=True)
```